tests/cmd/e2e/main.go

```diff
@@ -148,6 +149,23 @@ func main() {
 			TiDBTokenLimit:         1000,
 			PDLogLevel:             "info",
 		},
+		{
```

Comment:
Why the one pd test in e2e test and stability test is different ? this test may only test deploy TiDB cluster, so I think the e2e test did a lot of extra work.

Reply:
ditto